README.rst

```diff
@@ -161,14 +161,40 @@ You can also run them in parallel:
 Building Sphinx docs
 ~~~~~~~~~~~~~~~~~~~~
 
-``cd`` into the ``doc`` directory and run:
+Setup a Python environment with ``sphinx`` and ``sagemaker``:
+
+::
+
+    conda create -n sagemaker python=3
+    conda activate sagemaker
+    conda install sphinx
+    pip install sagemaker --user
+
+Install the Read The Docs theme:
+
+::
+
+    pip install sphinx_rtd_theme --user
```

Comment:
would it be worth adding a "doc" entry to https://github.com/aws/sagemaker-python-sdk/blob/master/setup.py#L47-L55?

Reply:
> would it be worth adding a "doc" entry to https://github.com/aws/sagemaker-python-sdk/blob/master/setup.py#L47-L55?

Mmmm no. IDK think it's a good idea to pop installing Sphinx on people. It has a lot of issues with versions and dependency incompatibilities. You could hose someone's setup by pushing a newer version of Sphinx and it's not critical to the core functionality of the sdk.

Reply:
neither `sphinx` nor `sphinx_rtd_theme` is in `setup.py` right now for what it's worth. @nadiaya do you have thoughts on this?

Reply:
I agree with Aaron, sphinx is not essential to any part of py-sdk functionality.